common/metrics/defs.go

```diff
@@ -2194,10 +2204,13 @@ const (
 
 	AsyncRequestPayloadSize
 
-	// emitted as timers to provide cluster-wide usage info.
-	// probably better as a histogram for better aggregation.
-	GlobalRatelimiterFallbackUsageTimer
-	GlobalRatelimiterGlobalUsageTimer
+	GlobalRatelimiterStartupUsageHistogram
+	GlobalRatelimiterFailingUsageHistogram
+	GlobalRatelimiterGlobalUsageHistogram
```

Comment:
split out "startup" because otherwise it'd be hard to build an automated alert, and moved to histogram because why not.

timers would be easy to use too.  bucket sizes are below.